IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -377,6 +378,17 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if([navigationAction navigationType] == WKNavigationTypeLinkActivated) {
+        //Open secure web links with target=new window in default browser
+        if(([requestURL.scheme.lowercaseString isEqualToString:@"https"] && !navigationAction.targetFrame.isMainFrame) ||
+           (![requestURL.scheme.lowercaseString hasPrefix:@"http"])) {  // Open non-web links with URL schemes that can be opened by the application
+            [MSIDAppExtensionUtil sharedApplicationOpenURL:requestURL];
```

Comment:
Let's add a log line here for debugging? (maybe info level)

Reply:
Addressed